### PR TITLE
#163588357 Update readme to include application setup

### PR DESCRIPTION
**Pull request by TooColline**
#### What does this PR do?
Update the README.md file

#### Description of Task to be completed?
- Update the file to include steps to clone the repository
- Update the file to include steps to set up the database
- Update the file to show how to run the server and the tests

#### How should this be manually tested?
1. Open your terminal and clone the repo with `https://github.com/andela/ah-bird-box.git`
2. Switch directory to the cloned project `cd ah-bird-box`
3. Checkout to the chore branch `git checkout ch-update-readme-163588357`
4. Create a virtual environment with `virtualenv -p python3 venv` or `python3 -m virtualenv venv`
5. Activate the virtual environment with `source venv/bin/activate`
6. Install the requirements packages with `pip3 install -r requirements.txt`

### Setting up the database
Ensure PostgreSQL is installed
1. Start the postgres database server using `pg_ctl -D /usr/local/var/postgres start`
2. Navigate to the psql terminal with `psql postgres`
3. Create a new user `CREATE USER sample_username WITH PASSWORD 'sample_password';`
4. Grant privileges to the user `ALTER USER sample_username CREATEDB;`
5. Create a database for the project `CREATE DATABASE sample_database_name WITH OWNER sample_username;`
6. Create a .env file under the main directory of the project and add:
```
source venv/bin/activate
export DATABASE_URL="postgresql://localhost:5432/<database_name>"
export EMAIL_SENDER="<your_email_address>"
export EMAIL_HOST="smtp.gmail.com"
export EMAIL_HOST_USER="<your_email_address>"
export EMAIL_HOST_PASSWORD="<your_email_password>"
export EMAIL_PORT=587
export FACEBOOK_KEY="<facebook_key_value>"
export FACEBOOK_SECRET="<facebook_secret_value>"
export GOOGLE_OAUTH2_KEY="<google_key_value>"
export GOOGLE_OAUTH2_SECRET="<google_secret_key>"
export OAUTH2_ACCESS_TOKEN="<oath_access_token>"
```
7. Do a `source .env` command to set the configuration variables
8. Make migrations to create tables in the database with `python manage.py makemigrations`
9. Run `python manage.py migrate`

### Run the server
Once everything is set up run the server using `python manage.py runserver`
The application can be accessed in the url http://localhost:800

#### Any background context you want to provide?
The configurations on the .env are meant to be updated with time
#### What are the relevant pivotal tracker stories?
[#163588357](https://www.pivotaltracker.com/n/projects/2240063)
#### Screenshots (if appropriate)
image

### CODE REVIEW

**Review by sam-karis — CHANGES_REQUESTED** [review]
Good work, just update the PR description to remove email credentials.

**Comment by TooColline** [comment]
Done.

File: README.md [hunk 1]
```diff
@@ -7,6 +7,56 @@ Create a community of like minded authors to foster inspiration and innovation
 by leveraging the modern web.
 
 ---
+## Application local setup
+
+### Technologies used to build the application
+1. [Django](https://www.djangoproject.com/)
+2. [Django Rest-Framework](https://www.django-rest-framework.org/)
+3. [Brew](https://brew.sh/)
```
Comment (sam-karis) [inline README.md]: I think `brew` is not used in this application

File: README.md [hunk 2]
```diff
@@ -7,6 +7,56 @@ Create a community of like minded authors to foster inspiration and innovation
...
+4. [PostgreSQL](https://medium.com/@Umesh_Kafle/postgresql-and-postgis-installation-in-mac-os-87fa98a6814d)
+5. [Virtualenv](https://virtualenv.pypa.io/en/latest/installation/)
+
+### Getting Started
+1. Open your terminal and clone the repo with `https://github.com/andela/ah-bird-box.git`
+2. Switch directory to the cloned project `cd ah-bird-box`
+3. Checkout to the chore branch `git checkout ch-update-readme-163588357`
```
Comment (sam-karis) [inline README.md]: To Setup the app you don't need to checkout to this branch because after it's merged it will be deleted

**Review by sam-karis — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (TooColline) [reply README.md]: Fixed that.

File: README.md (hunk 2 above)
Comment (TooColline) [reply README.md]: Noted and fixed.

**Review by sam-karis — APPROVED** [review]

**Pull request merged**